### PR TITLE
MAINTAINERS: cleanup x86/areas areas

### PR DESCRIPTION
**Pull request by nashif**
cleanup x86/xtensa areas and set new maintainers.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by teburd — DISMISSED** [review]

**Comment by teburd** [comment]
I'm fine being a collaborator here, and makes sense to me

**Review by edersondisouza — DISMISSED** [review]

**Review by jhedberg — DISMISSED** [review]

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -1876,6 +1868,8 @@ Release Notes:
     - teburd
   files:
     - drivers/timer/
+    - tests/kernel/timer/
+    - tests/drivers/timer/
```
Comment (MaureenHelm) [inline MAINTAINERS.yml:1872]: This looks unrelated

Comment (nashif) [reply MAINTAINERS.yml:1872]: it is, but does this now justify another PR to this non-code-changing PR?

Comment (MaureenHelm) [reply MAINTAINERS.yml:1872]: another PR no, but another commit would be nice

**Review by jhedberg — APPROVED** [review]

**Review by ceolin — APPROVED** [review]

**Review by edersondisouza — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]
